src/fetch_package.js

```diff
@@ -121,25 +133,26 @@ export default async function fetchPackage(
 
         /**
          * We're caching.
-         *
-         * Set the expiration of the cache at 900 seconds (15 minutes).
-         * This feels reasonable for now.
          */
-        return fetcher
-            .use(superagentCache)
-            .expiration(900)
-            .prune((response, gutResponse) => {
-                /**
-                 * We want to use our own `prune` method so that we can track
-                 * what comes from cache versus what doesn't.
-                 *
-                 * But we still do the same thing that superagent-cache would
-                 * do, for now.
-                 */
-                const guttedResponse = gutResponse(response);
-                guttedResponse._token = token;
-                return guttedResponse;
-            });
+        return (
+            fetcher
+                .use(superagentCache)
+                // Set the expiration of the cache to be really high (24 hours)
+                // as the files aren't expected to ever change.
```

Comment:
SUPER NIT: Tend to use `/**/` block comments for multiline comment blocks like this.